.circleci/config.yml

```diff
@@ -345,17 +352,26 @@ on-version-tags: &on-version-tags
       only: /^v[0-9].*/
 
 on-master-and-version-tags: &on-master-and-version-tags
+  filters:
+    branches:
+      only: master
+    tags:
+      only: /^v[0-9].*/
```

Comment:
remove `on-master-and-version-tags` as it is no longer in use, or combine both definitions

Reply:
yes.